WordPress/Classes/Models/ReaderDefaultTopic.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+@objc public class ReaderDefaultTopic : ReaderAbstractTopic
+{
+    public class var TopicType: String {
```

Comment:
Oops. TopicType isn't declared in the base class otherwise those would have to be overridden.